tests/test_actuators.py

```diff
@@ -21,7 +21,7 @@ def test_actuator(self):
         with self.assertRaises(NotImplementedError):
             a.serialize()
         with self.assertRaises(NotImplementedError):
-            a.load()
+            a.load({})
```

Comment:
I was focused on the type and saw no value in allowing None. An empty dict is at least type-wise OK. Content-wise, not so much, but validation of the serialization container is definitely over the scope of this issue.
BTW, the base class defines load(self, data) as an instance method but the descendant classes define static @classmethods with the same name instead of overrides. That needs a cleanup, too - also not in the scope of this issue.

Reply:
Your previous point is excellent and you are right obviously on the None and Optional thing.
I'll look at the `@classmethod` issue, it sounds like something I could have done... Late coding is not always the best coding...